interfaces/Config/templates/config_notify.tmpl

```diff
@@ -363,6 +363,49 @@
            </fieldset>
        </div>
     </div>
+    <div class="section" id="apprise">
+        <div class="col2">
+            <h3>$T('section-Apprise')</h3>
+            <table>
+                <tr>
+                    <td><input type="checkbox" name="apprise_enable" id="apprise_enable" value="1" <!--#if int($apprise_enable) > 0 then 'checked="checked"' else ""#--> /></td>
+                    <td><label for="apprise_enable"> $T('opt-apprise_enable')</label></td>
+                </tr>
+            </table>
+            <em>$T('explain-apprise_enable')</em>
+            $show_cat_box('apprise')
+        </div>
+        <div class="col1" <!--#if int($apprise_enable) > 0 then '' else 'style="display:none"'#-->>
+            <fieldset>
+                <div class="field-pair">
+                    <label class="config" for="apprise_urls">$T('opt-apprise_urls')</label>
+                    <input type="text" name="apprise_urls" id="apprise_urls" value="$apprise_urls" />
+                    <span class="desc">$T('explain-apprise_urls')</span>
+                </div>
+					 <hr />
+                <span class="desc">$T('explain-apprise_extra_urls')</span>
+                <!--#set $section_label = 'apprise'#-->
+                <!--#for $type in $notify_types#-->
+                    <div class="field-pair">
+                        <label class="config" for="${section_label}_target_${type}">
+                            $T($notify_types[$type]).replace('/', ' / ')
+                        </label>
+                        <input type="checkbox" name="${section_label}_target_${type}_enable" id="${section_label}_target_${type}_enable" value="1" <!--#if int($getVar($section_label + '_target_' + $type + '_enable')) > 0 then 'checked="checked"' else ""#--> />
+
+                        <input type="text" name="${section_label}_target_${type}" id="${section_label}_target_${type}" value="$getVar($section_label + '_target_' + $type)" />
```

Comment:
Let's add a `placeholder="$T('default')"` to the overrides input.

Reply:
Done :slightly_smiling_face: